docs/user_manual/components.md

```diff
@@ -18,7 +18,7 @@ The base type for all power-grid-model components.
 
 | name | data type | unit | description                                                                                                                                                       | required |                                     update                                     |
 | ---- | --------- | ---- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------: | :----------------------------------------------------------------------------: |
-| `id` | `int32_t` | -    | ID of a component. The ID should be unique across all components within the same scenario, e.g., you cannot have a node with `id=5` and another line with `id=5`. | &#10004; | &#10060; (id needs to be specified in the update query, but cannot be changed) |
+| `id` | `int32_t` | -    | ID of a component. The ID should be unique across all components within the same scenario, e.g., you cannot have a node with `id=5` and another line with `id=5`. | &#10004; | &#10060; (If a component update is uniform, the ID can be omitted or set to `nan`. Otherwise, the id needs to be specified in the update query, but cannot be changed) |
```

Comment:
i don't know whether a user understands what `uniform` means if you put it in the components like this.

you can also just say something like `(see below)` and then have a note block elaborate on the topic

Reply:
Good idea, I will add a block further explaining below.